src/project/name.js

```diff
@@ -28,6 +28,12 @@ const getProductName = (projectPath) => {
       const viaPath = `${projectPath}/.idea`;
       const imlFiles = getFiles(viaPath).filter(file => path.extname(file) === '.iml');
       return (imlFiles.length === 1) ? path.basename(imlFiles[0], '.iml') : false;
+    },
+    viaDirectoryName: () => {
+      if (!isIdeaDirExists) {
```

Comment:
this test is required for that case?

Reply:
Debatable. An existing project should always have this folder, but one could just `rm -rf .idea`. If opened through IntelliJ afterwards, it would just be recreated. Since the project list is sourced from IntelliJ's xml files, those should be already valid projects. So I'm now leaning on removing it, but this should definitely remain the last case to evaluate.

Reply:
Thanks for the explanation, I didn't see that case